### ISSUE TITLE
repoquery --deplist

### ISSUE THREAD

**Issue by j-mracek** [rpm-software-management/dnf5#914]
The option is also an alias in DNF4 for deplist command

[amatej] Perhaps this would be better suited as a plugin?

**Comment by m-blaha** [comment]
Yes, I support a plugin solution if there is demand for one. I believe a single `deptree` plugin could cover the functionality of both `repoquery --tree` (see https://github.com/rpm-software-management/dnf5/issues/913) and `repoquery --deplist` (which is essentially a depth-one slice of the dependency tree traversal).
